tests/test_jobs.py

```diff
@@ -484,69 +501,25 @@ def test_resuming(self):
         expected = ['4ab289ab60aee93a66e5490529799cf4a2b4d94c']
         self.assertEqual(issues, expected)
 
-    def test_initialize_cache(self):
-        """Test if the cache is initialized"""
+    def test_initialize_archive_manager(self):
+        """Test if the archive is initialized"""
 
-        job = PercevalJob('arthur-job-1234567890', 'mytask', 'git',
+        job = PercevalJob('arthur-job-1234567890', 'mytask', 'git', 'commit',
                           self.conn, 'items')
-        job.initialize_cache(self.tmp_path)
 
-        self.assertIsInstance(job.cache, Cache)
-        self.assertEqual(job.cache.cache_path, self.tmp_path)
+        self.assertIsNone(job.archive_manager)
```

Comment:
I think it would be a better idea to check whether the manager is an instance of `ArchiveManager`.